client/scripts/views/pages/discussions/index.ts

```diff
@@ -63,23 +63,42 @@ const getLastSeenDivider = (hasText = true) => {
   );
 };
 
+
 export const CommunityOptionsPopover: m.Component<
   { isAdmin: boolean; isMod: boolean },
   {}
 > = {
   view: (vnode) => {
     const { isAdmin, isMod } = vnode.attrs;
     if (!isAdmin && !isMod && !app.community?.meta.invitesEnabled) return;
+
+    // add extra width to compensate for an icon that isn't centered inside its boundaries
+    const DropdownIcon = m('div', {
```

Comment:
Typically, we don't just write "div" for a component names, we try to use a more "human-readable" class name which can be something like `.dropdown-spacing` or `.dropdown-wrapper`. This will automatically make the mithril component be a div, but it'll be classed something we can quickly understand.